contract/src/platform-goals/start-governed-contract.js

```diff
@@ -1,281 +0,0 @@
-import { E } from '@endo/far';
```

Comment:
if we're getting rid of this, we can drop some other stuff such as the dependency on `@agoric/governance`.